<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskOverViewItem</class>
 <widget class="QWidget" name="TaskOverViewItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TaskOverViewItem</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="taskOverItemContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 255, 127,200);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>任务时间:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>任务名称:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>2020年1月20日-2020年1月31日</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>XX任务</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>任务地点:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>南海</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
